internal/backend/rclone/stdio_conn.go

```diff
@@ -0,0 +1,72 @@
+package rclone
```

Comment:
It ocurred to me that this is the kind of useful little object that might go in its own repo.  Rclone could then import it too!

Reply:
That also occurred to me, but I prefer to not couple the source code too much for now. We can always move it out later (ideally, when `vgo` is merged)